harmonica/forward/point_mass.py

```diff
@@ -231,30 +240,61 @@ def kernel_potential_cartesian(
     easting, northing, upward, easting_p, northing_p, upward_p
 ):
     """
-    Kernel function for potential gravity field in Cartesian coordinates
+    Kernel function for potential gravitational field in Cartesian coordinates
     """
-    return 1 / distance_spherical(
-        (easting, northing, upward), (easting_p, northing_p, upward_p)
+    distance = distance_cartesian(
+        [easting, northing, upward], [easting_p, northing_p, upward_p]
```

Comment:
Is there a special reason why you replaced tuples with lists?
We try to avoid passing lists to Numba jitted functions because Numba has deprecated list reflection (see #70 and [this Numba doc entry](https://numba.pydata.org/numba-doc/latest/reference/deprecation.html?highlight=reflected#deprecation-of-reflection-for-list-and-set-types) for more information).
Is there any deprecation warning raise when running the tests?

Reply:
I just followed the same notation used in other functions of `point_mass.py`. There was no deprecation warning raise.

Reply:
Ok. So, I don't know why passing some lists to Numba jitted functions seem to break things and with other list it has no trouble. I would change everything to tuples to prevent possible headaches in the future regarding lists. It seems that when Numba tries to work with lists (which are mutable), they need some special care because of their more complex structure when compared with tuples. What do you think?

Reply:
Ok for me @santisoler . I can change all list to tuples.

Reply:
I have replaced all lists by tuples to avoid problems with Numba